compiler/src/iree/compiler/InputConversion/Common/QuantizedMatmulToMatmul.cpp

```diff
@@ -88,34 +114,41 @@ struct QuantizedMatmulToMatmul
     int indexOfLhsZpTimesRhsZpTimesKSizeInput = 0;
     Type accElTy = accType.getElementType();
     if (!rhsZpIsConstantZero) {
+      SmallVector<bool> colRedIterator = {false, true};
+      if (hasBatchDim)
```

Comment:
Please add `{...}` braces for all control flow in line with the existing code around here. I know our coding style allows omitting some braces, but consistency with the surrounding code matters, as omitted braces can be surprising.

Reply:
Yes, it makes sense to have braces around the non-obvious/intricate parts but I have omitted them in the obvious part i.e., for checking the failure condition.